### DISCUSSION TITLE
Website for business

### DISCUSSION THREAD

**Discussion by ellielo** [NAIST-SE/AttractingContributors#337] [General]
Hi guys !
What is the easiest way to create a website from scratch?
THanks

**Comment by hernessteve** [comment]
You can find a freelancer developer on Upwork or Fiverr. One thing is to create a website and more difficult task is to attract people on your website.

**Comment by MirkosMangle** [comment]
Managing business with all the modern technologies available now has become significantly easier. It's fascinating how even churches are leveraging separate texting services, as detailed here [https://clerk.chat/industries/church/](https://clerk.chat/industries/church/) These tools streamline communication within congregations, facilitate event coordination, and enhance outreach efforts. Whether in business or religious organizations, integrating such technology not only improves efficiency but also strengthens community engagement

**Comment by XelaAl** [comment]
Creating a website from scratch can be easier than it seems! Platforms like WordPress or Wix offer user-friendly templates. For a more custom approach, consider learning HTML and CSS. I recently came across a study on touch-based user interfaces https://dworkz.com/article/touch-based-user-interfaces-usability-study/, and it really emphasizes how essential intuitive design is for user experience. It’d be great if that’s something you consider when building websites.
